### PR TITLE
docs(time-series-chart): rewrite stories in new format

### PR DESCRIPTION
**Pull request by amevck**

### CODE REVIEW

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-557.gearboxjs.preview.cogniteapp.com

File: src/components/TimeseriesChart/stories/basic.story.mdx [hunk 1]
```diff
@@ -0,0 +1,352 @@
...
+
+See more details in `Custom container styles` example.
+
+## Examples
+
+### Empty 
+
+<Preview>
+  <Story name="Empty" >
+   <TimeseriesChart timeseriesIds={[]} />
```
Comment (mrtraser) [inline src/components/TimeseriesChart/stories/basic.story.mdx]: alignment

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-557.gearboxjs.preview.cogniteapp.com

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-557.gearboxjs.preview.cogniteapp.com

**Pull request merged**